governance/polls/Add GUNIV3DAIUSDC2-A with 0.01% Fee Tier as a new Vault Type - November 29, 2021.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Add GUNIV3DAIUSDC2-A with 0.01% Fee Tier as a new Vault Type - November 29, 2021
+summary: This poll proposes system parameters which could be used to initialize GUNI-DAI-USDC-2-A as a new vault type.
+discussion_link: https://forum.makerdao.com/t/g-uni-dai-usdc-0-01-tier-onboard-g-uni-for-univ3s-new-0-01-fee-tier/11738
+vote_type: Plurality Voting
+categories:
+   - Collateral
+   - Risk Variable
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2021-11-29T16:00:00
+end_date: 2021-12-02T16:00:00
+---
+# Poll: Add GUNIV3DAIUSDC2-A with 0.01% Fee Tier as a new Vault Type - November 29, 2021
```

Comment:
I don't think you need "with 0.01% Fee Tier" here

Reply:
fxd!